messagease/en-us.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<keyboard bottom_row="false" width="6.0" name="MessagEase (US)" script="latin">
+  <row>
+    <key shift="1" key1="esc" key3="config"/>
+    <key key0="a" key1="“" key2="”" key3="$" key4="v" key5="—" key6="-" key7="–"/>
+    <key key0="n" key1="`" key2="´" key3="/" key4="\\" key5="+" key6="!" key7="^" key8="l"/>
+    <key key0="i" key1="‘" key2="’" key3="x" key4="€" key5="\?" key8="="/>
+  </row>
+  <row>
+    <key shift="1" key0="switch_numeric" key1="selectAll" key2="cut" key4="pasteAsPlainText" key5="switch_backward" key6="switch_forward" key7="copy" key8="paste"/>
+    <key key0="h" key1="{" key2="%" key3="[" key4="_" key5="(" key6="k"/>
+    <key key0="o" key1="q" key2="p" key3="g" key4="j" key5="c" key6="b" key7="u" key8="d"/>
+    <key key0="r" key1="|" key2="}" key3="\@" key4="]" key5="m" key6=")" key7="shift" key8="capslock"/>
+  </row>
+  <row>
+    <key shift="1" key0="backspace" key6="delete" key8="insert"/>
+    <key key0="t" key1="~" key2="y" key3="«" key4="tab" key5="&lt;" key6="*"/>
+    <key key0="e" key1="&quot;" key2="'" key3="," key4=":" key6="z" key7="w" key8="."/>
+    <key key0="s" key1="f" key2="°" key3=";" key4="»" key5="\#" key6=">" key7="&amp;" key8="…"/>
+  </row>
+  <row>
+    <key shift="1" key0="enter" key2="switch_emoji" key3="action"/>
+    <key width="3" key0="space" key1="alt" key2="ctrl" key4="nbsp" key5="left" key6="right" key7="up" key8="down"/>
```

Comment:
You can set `slider="true"` to benefit from the slider gesture for left and right.

Reply:
D’oh. I think I just edited that in on the phone but forgot to add it to the version I pasted here.

Also, for some reason, `width="6.0"` on the root element seems to have no effect for me; I expected to have a key’s worth of padding on both sides but only getting padding on the left due to `shift="1"`.